src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -32,11 +32,54 @@ public sealed class DotNetMSBuildSdkResolver : SdkResolver
         private readonly Func<string, string, string?> _getMsbuildRuntime;
         private readonly NETCoreSdkResolver _netCoreSdkResolver;
 
+        private const string DOTNET_HOST = nameof(DOTNET_HOST);
         private const string DotnetHostExperimentalKey = "DOTNET_EXPERIMENTAL_HOST_PATH";
         private const string MSBuildTaskHostRuntimeVersion = "SdkResolverMSBuildTaskHostRuntimeVersion";
-
+        private const string SdkResolverHonoredGlobalJson = "SdkResolverHonoredGlobalJson";
+        private const string SdkResolverGlobalJsonPath = "SdkResolverGlobalJsonPath";
         private static CachingWorkloadResolver _staticWorkloadResolver = new();
 
+        /// <summary>
```

Comment:
@dsplaisted I've made that change we talked about now - want to check me? I did load the updated MSBuild dll in an FSI session and made sure that the reflection-based lookup below does in fact resolve.